bsyncviewer/settings/docker_dev.py

```diff
@@ -0,0 +1,17 @@
+"""
+Django settings for bsyncviewer project.
+
+Generated by 'django-admin startproject' using Django 2.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+from bsyncviewer.settings.docker import *  # noqa: F401, F403
+
+DEBUG = True
+
```

Comment:
Im not sure if this is just something on my end but I had to add `ALLOWED_HOSTS = ['0.0.0.0']` to this file.